packages/transformers/json/src/JSONTransformer.js

```diff
@@ -5,15 +5,22 @@ import json5 from 'json5';
 
 export default (new Transformer({
   async transform({asset}) {
-    asset.type = 'js';
-    // Use JSON.parse("...") for faster script parsing, see
-    // https://v8.dev/blog/cost-of-javascript-2019#json.
-    // Apply `JSON.stringify` twice to make it a valid string literal.
-    asset.setCode(
-      `module.exports = JSON.parse(${JSON.stringify(
-        JSON.stringify(json5.parse(await asset.getCode())),
-      )});`,
-    );
+    // This indicates a previous transformer (e.g. WebExt) has applied special
+    // handling to this already
+    if (asset.meta.handled) {
```

Comment:
Why is this needed, just for pretty formatting? In that case we could make `raw:...` "pretty" be default

Reply:
If we don't do this, we don't allow users to have files named `manifest.json` that aren't web extension manifests.